lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -168,12 +168,7 @@ def get(self, request, *args, **kwargs):
         offer_html = show_enrolled and generate_offer_html(request.user, course_overview)
         course_expired_html = show_enrolled and generate_course_expired_message(request.user, course_overview)
 
-        welcome_message_html = None
-        if show_enrolled:
-            if LATEST_UPDATE_FLAG.is_enabled(course_key):
-                welcome_message_html = LatestUpdateFragmentView().latest_update_html(request, course)
-            elif get_course_tag(request.user, course_key, PREFERENCE_KEY) != 'False':
-                welcome_message_html = WelcomeMessageFragmentView().welcome_message_html(request, course)
+        welcome_message_html = show_enrolled and get_current_update_for_user(request, course)
```

Comment:
So this change is because:
- I moved the `get_course_tag` checking down a level, to the new utility methods.
- I realized that the LATEST_UPDATE_FLAG doesn't actually change the *content* of what the user sees -- it is an old (2017/2018) opt-out config for certain courses (all mitxpro) to change two things: updates aren't dismissable and have a different labeling (they have a "Latest Update" header with the flag on). Since the MFE doesn't really support that, and we probably don't care to(?), I just dropped checking the flag here.